website/content/docs/connect/cluster-peering/create-manage-peering.mdx

```diff
@@ -0,0 +1,138 @@
+---
+layout: docs
+page_title: Create and Manage Peering Connections
+description: >-
+    This page describes how to use the Consul CLI to create, manage, and delete peering connections for cluster peering.
+---
+
+# Create and Manage Peering Connections
+
+~> This page covers features that are currently in _technical preview_. Features and functionality are subject to change. You should never use the technical preview release in secure environments or production scenarios. Features in technical preview may face performance and scaling issues, with limited support options available.
+
+A peering token enables cluster peering between different datacenters. Once you generate a peering token, you can use it to initiate a connection between clusters. Then you can export services and authorize other clusters to call those services.
+
+To peer clusters, you must complete the following steps in order:
+
+1. Create a peering token
+1. Establish a connection between clusters
+1. Connect service endpoints
+1. Authorize connections between peers
+
+## Create a peering token
+
+You can generate peering tokens and initiate connections from either the server agents or the client agents in your clusters. For the current release, we recommend you initiate peering through the client agents in the partitions you want to connect.
+
+To begin the cluster peering process, generate a peering token in one of your clusters. The other cluster uses this token to establish the peering connection.
+
+In “cluster-01,” run the `generate-token` command.
+
+```shell-session
+$ consul peering generate-token -peer="cluster-02"
+```
+
+The CLI outputs the peering token, which is an encoded string of alphanumeric characters and symbols.
+
+For the peering token to function correctly, you must enter the second cluster’s exact name in the `peer` parameter.
+
+## Establish a connection between clusters
+
+Next, use the peering token to establish a secure connection between the clusters. In the client agents of “cluster-02,” run the ``peering establish`` command. This command does not generate an output.
+
+```shell-session
+$ consul peering establish -peer="cluster-01" -token="eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJhZG1pbiIsImF1ZCI6IlNvbHIifQ.5T7L_L1MPfQ_5FjKGa1fTPqrzwK4bNSM812nW6oyjb8"
+```
+
+In the peer parameter, specify the first cluster’s exact name. The `token` parameter should include the entire peering token created in the first cluster.
+
+When you connect server agents through cluster peering, they will peer through their default partitions. To connect other partitions through server agents, you must specify the partitions you want to peer using the `partition` parameter.
```

Comment:
Some clarifications since peering is a relationship between partitions not something that happens through partitions:

```suggestion
When you connect server agents through cluster peering, they will peer their default partitions. To establish peerings for other partitions through server agents, you must specify the partitions you want to peer using the `Partition` field of the request body.
```